.github/workflows/scripts/smoke-test-eventing.sh

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env bash
+
+# The Knative Eventing webhook occasionally responds with "connection refused"
+# although its Pod(s) reports "Ready", making Knative Eventing unusable.
+# This script performs a smoke test of the Eventing installation by attempting
+# a create/delete cycle of a basic Channel object. In case of failure, it
+# deletes the Eventing webhook Pod(s) and retries.
+
+set -eu
+set -o pipefail
+
+function create_channel {
+	local err
+
+	err="$(kubectl create -f- 2>&1 >/dev/null <<-EOM
```

Comment:
Same here: `-f -`

Reply:
Addressed in [4edd5801..5d40b9c2](https://github.com/triggermesh/triggermesh/compare/4edd580109b4f6f20dcce598819e3b5a23f9be6b..5d40b9c2eba88353589183cf6b8fd1be281db44c)